vue.config.js

```diff
@@ -9,6 +9,25 @@ const aboutText1 = (appName && appVersion) ? `${appName} v${appVersion}` : ''
 const aboutText2 = (sbcName && sbcVersion) ? `${sbcName} v${sbcVersion}` : ''
 
 module.exports = {
+  chainWebpack: (config) => {
+    // alias vue to @vue/compat
+    config.resolve.alias.set('vue', '@vue/compat')
+    // enable compat mode via Vue compiler options
+    config.module
+      .rule('vue')
+      .use('vue-loader')
+      .tap((options) => {
+        return {
+          ...options,
+          compilerOptions: {
+            compatConfig: {
+              MODE: 2
+            }
+          }
+        }
+      })
+  },
```

Comment:
This config was recommended by the Vue3 migration side ([**step 3**](https://v3-migration.vuejs.org/migration-build.html#upgrade-workflow)).

If we go pure Vue3 then we probably don't need this config. To be determined later.